<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RK txt </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>721</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help_pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Справка</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnRU">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Русский</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnENG">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openRead_pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть для чтения</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
